prime-router/settings/organizations-local.yml

```diff
@@ -340,7 +348,7 @@
           receivingFacilityName: NMDOH
           receivingFacilityOID: 2.16.840.1.113883.3.5364
           truncateHDNamespaceIds: true
-          nameFormat: APHL_LIGHT
+          nameFormat: CUSTOM
```

Comment:
Consider putting the nameFormat field inside of the fileName: section.    Or get rid of it completely?   If the fileName: section exists, it overrides the default fileName.

Also, please consider changing the name from fileName to fileNameTemplate.   I think of 'filename' as an actual filename string, not the template to create the filename string.

Reply:
Unfortunately I won't be able to move this easily because `nameFormat` is in the JSON objects in the `Setting` table and removing it entirely will break stuff unless I create a database migration for that. I'm not opposed to it. I'd use the same time to remove the `useAphlNameFormat` field as well. So I will probably do this.